tests/compatibility-test.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+import logging
+import os
+import sys
+import tempfile
+import time
+import unittest
+from string import Template
+
+import docker
+
+ray_version = '1.8.0'
+ray_image = "rayproject/ray:1.8.0"
+
+kindcluster_config_file = 'tests/config/cluster-config.yaml'
+
+raycluster_spec_template = 'tests/config/ray-cluster.mini.yaml.template'
+raycluster_service_file = 'tests/config/raycluster-service.yaml'
```

Comment:
I can not find the usage of this file. What's the usage here?

Reply:
raycluster-service.yaml was the same as the one from patch #147